package.json

```diff
@@ -8,10 +8,30 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "seed": "node ./server/models/seed.js",
+    "start": "nodemon server/index.js",
```

Comment:
Your `start` script should probably not run `nodemon` given you are not actively coding in production.